apps/studio/components/interfaces/DiskManagement/ui/DiskSpaceBar.tsx

```diff
@@ -187,16 +201,18 @@ export default function DiskSpaceBar({
           )}
         </AnimatePresence>
       </div>
-      <div className="flex items-center space-x-4 text-xs text-foreground-lighter">
-        <div className="flex items-center">
-          <div className="w-2 h-2 rounded-full bg-foreground mr-2" />
-          <span>Used Space</span>
-        </div>
-        <div className="flex items-center">
-          <div className="w-2 h-2 rounded-full bg-border border border-strong mr-2" />
-          <span>Available space</span>
+      {!show && (
+        <div className="flex items-center space-x-4 text-xs text-foreground-lighter">
+          <div className="flex items-center">
+            <div className="w-2 h-2 rounded-full bg-foreground mr-2" />
+            <span>Used Space</span>
+          </div>
+          <div className="flex items-center">
+            <div className="w-2 h-2 rounded-full bg-border border border-strong mr-2" />
+            <span>Available space</span>
+          </div>
         </div>
-      </div>
+      )}
```

Comment:
![screenshot-2024-10-30-at-15 02 31](https://github.com/user-attachments/assets/68997166-b0b2-4dd5-bd55-57ac4435bc57)

Reply:
yea probably. I'll add it after - don't want to break anything.